tests/integrationtest/r/planner/core/casetest/partition/partition_pruner.result

```diff
@@ -928,3 +928,19 @@ Point_Get	1.00	root	table:t, partition:p4, index:idx_3(col_4)
 select * from t where col_4 in ( 'u^D92@_4' ,null );
 col_4
 u^D92@_4
+drop table if exists t;
+CREATE TABLE `t` (
+`col_95` char(181) COLLATE gbk_bin NOT NULL DEFAULT 'SaMKHTyg+nlID-X3Y',
+PRIMARY KEY (`col_95`) /*T![clustered_index] CLUSTERED */
+) ENGINE=InnoDB DEFAULT CHARSET=gbk COLLATE=gbk_bin
+PARTITION BY RANGE COLUMNS(`col_95`)
```

Comment:
Do we need to test other partitioning schemes as well? Like RANGE COLUMNS, LIST [COLUMNS] and KEY?

Reply:
I don't think this is related to partition types. And the test cases have already been added in [PR #59918](https://github.com/pingcap/tidb/pull/59918).